cmd/vigilante/cmd/btcstaking_tracker.go

```diff
@@ -61,26 +60,17 @@ func GetBTCStakingTracker() *cobra.Command {
 
 			// create BTC client and connect to BTC server
 			// Note that monitor needs to subscribe to new BTC blocks
-			btcClient, err := btcclient.NewWithBlockSubscriber(
-				&cfg.BTC,
-				cfg.Common.RetrySleepTime,
-				cfg.Common.MaxRetrySleepTime,
-				rootLogger,
-			)
+			btcClient, err := btcclient.NewWallet(&cfg.BTC, rootLogger)
+
 			if err != nil {
 				panic(fmt.Errorf("failed to open BTC client: %w", err))
 			}
 
 			// create BTC notifier
 			// TODO: is it possible to merge BTC client and BTC notifier?
-			btcParams, err := netparams.GetBTCParams(cfg.BTC.NetParams)
+			btcNotifier, err := btcclient.NewNodeBackendWithParams(cfg.BTC, "")
 			if err != nil {
-				panic(fmt.Errorf("failed to get BTC parameter: %w", err))
-			}
-			btcCfg := btcclient.CfgToBtcNodeBackendConfig(cfg.BTC, "") // we will read certifcates from file
-			btcNotifier, err := btcclient.NewNodeBackend(btcCfg, btcParams, &btcclient.EmptyHintCache{})
-			if err != nil {
-				panic(fmt.Errorf("failed to create btc chain notifier: %w", err))
+				panic(err)
```

Comment:
```suggestion
				panic(fmt.Errorf("failed to create btc chain notifier: %w", err))
```

Reply:
errors are already wrapped in the `NewNodeBackendWithParams`